source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+
+[subscription].[SubscriberAccounts]
+---------------------------------
+Contains the first date for when a SubscriberAccount was used. This is used for financial reporting.
+
+[subscription].[SubscriptionPackageProducts]
+--------------------------------------------
+All subscription packages related to product.
+
+[subscription].[SubscriptionPackages]
+--------------------------------------------
+All personal subscription packages for a subscriber. Can be used on several subsequent subscriptions. Is basically a calculated copy of the template package.
+
+Special fields in SubscriptionPackages:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**BillingFrequencyId**: Example of values 1001 = Month, 1003 = Quarter, 1012 = Full year.
+
+**BillingPlanId**: Each term can be linked to several payment plans . A payment plan controls, for example, how early payment claims are to be formed and when they are to be reminded. For example , you may want a 30-day payment deadline for companies, but three weeks for regular subscribers.
+
+**SubscriptionPackageChainId**: This is used if you create package links. That is, you must first pay 99 kroner the first month, then 149 and maybe 199 before you go over to full price 249.
+
+**InitialTermType**: This is used if the first period is to have a different length. 10 = «Until date », 20 = «Number of days», 100 = «Out the month», 200 = «Out the year» .  
+
+**InitialTermValue**: This will then have slightly different values ​​depending on the type. For the value «10», then there will be a date. For "20" it will be a number of days ". For "100" and "200", it is not used.
+
+**AutomaticStop**: This means that the subscription will be automatically stopped after the period.
+
+ 
+[subscription].[Subscriptions]
+------------------------------
+All subscriptions registered in the system, with comprehensive information about, among other things, start and end time, details such as price, tax ( VAT ) , units, IsCancelled (0 = No), IsRenewed (0 = no) and reason for cancellation.
+
+ 
+
+
+
+
+
+Appendix
+========
+
+To get a better overview of the tables in the reporting database, you can use the SQLs below as an aid.
+ 
+
+Count the number of columns in all tables
+-----------------------------------------
+
+select count ( COLUMN_NAME ) CountColumns from INFORMATION_SCHEMA . COLUMNS where TABLE_NAME not in ( '__EFMigrationsHistory' , 'Snapshots' , 'PowerBiConfigurations' , 'Commits' ) and TABLE_SCHEMA <> 'Sys'
+
+
+Output per. 20 November 2020: 446
+
+Lists all columns in all tables
+-------------------------------
+
+select TABLE_SCHEMA + '.' + TABLE_NAME as ' TableSchema.Table_Name ' , max ( ORDINAL_POSITION ) as ColumnsCount
+from INFORMATION_SCHEMA . COLUMNS where   TABLE _NAME not in ( '__EFMigrationsHistory' , 'Snapshots' , 'PowerBiConfigurations' , 'Commits' ) and TABLE_SCHEMA <> 'Sys'
+group by TABLE_SCHEMA + '.' + TABLE_NAME order by TABLE_SCHEMA + '.' + TABLE_NAME
+
+.. image:: Tables.png
```

Comment:
.. image:: /_images/Tables.png
    :align: center
    :alt: Example list of all accessible tables in the reporting database.

Reply:
Updated!